config/routes.rb

```diff
@@ -1,6 +1,14 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  root "auth#signup"
 
-  # Defines the root path route ("/")
-  # root "articles#index"
+  get '/', to: 'auth#signup'
+  get 'signup', to: 'auth#signup'
+  get 'landing', to: 'auth#landing'
+  get 'home', to: 'auth#home'
+  get 'login', to: 'auth#login'
+
+
+  post "/sessions", to: "auth#login"
+  post "/users", to: "auth#signup"
+  post "/delete/session", to: "auth#logout"
```

Comment:
This should not be a POST request to `/delete/session` it should be a DELETE request to `/sessions`.